.github/labels.yml

```diff
@@ -32,3 +29,30 @@
 - name: enhancement
   description: adds a new feature or improves on an existing one
   color: ad12cc
+- name: dependencies
+  description: Update one or more dependencies version
+  color: 8732bc
```

Comment:
Can we keep the original colour and description for this label.

Reply:
:eyes:

Reply:
Sorry, put the colour back but not the description 🤦

Reply:
done now.